_posts/2023-10-21-beneath-the-shell.md

```diff
@@ -0,0 +1,65 @@
+
+## Early concepts: the "super-sizing"
+
+The story begins over two centuries before the first ka'het saw the light; not when the proposal was first conceived (the idea itself is much older, and always rejected as outlandish and unfeasible), but when it was first realized, through a tech demo that applied what is known as "super-sizing" an organism and created a three meter long, half a meter large worm, incapable of sustaining its own weight in a gravitational field or survive on its own. Though its appearance would be much different from what followed, and it had no functions beyond eating and growing, nothing would have been possible without the breakthroughs of the super-sizing project - as a matter of fact, it wasn't long after this that the Fetri'sei program began to make its first steps.
+
+<img class="centered" src="/images/blog/ka-het/super-sized.jpg" width="400" height="300" />
+
+## The Fetri'sei, and the exploration of deep space
+
+It was around this time that, between the various departments of the interstellar authority, the Ka'sei were nearing their peak: managing over two dozen stations in the core systems (with three more in various stages of construction) and a number smaller projects on the surface, they had grown far beyond the role they were supposed the have and made many inside the government unease about the power they could exert. To start breaking the hegemony that was starting to form the Ka'mar were created, intended to take some of the most promising space-based projects and develop them to completion - for instance, the then-new program of automating a large part of the interstellar traffic through new, robotic spacecrafts and surface bases. The Ka'sei's response didn't have to wait long: a competing program, which instead of automated spacecrafts put super-sized organisms at the helm and would have been capable of space travel, exploration and self replication; thus, the Fetri were born.
+
+<img class="centered" src="/images/blog/ka-het/fetri-sei-exploded.jpg" width="570" height="370" />
+
+With a lifetime measured in hundreds of years and an essentially infinite capability to replicate, it could achieve something only a few probes could ever do: exploring what's beyond the hyperlanes, going further than ever before at sublight speeds and report back everything that can be seen. Which it did, in some way. Nearly a century after the program's creation the first Ka'sei reached a neighbouring system, a g-type star disconnected from the hyperlane net, and was blown apart within seconds; the same happened to its three sister probes, standing a few days behind the first, in such a precise and constant way that only a sentient being could. The first result was panic; the second, however, was the clear realization of the need for a way to defend from the rest of the universe, and strike back to teach them their place, much more than simple warships could. The situation was dire, however.
```

Comment:
> I think this could use some clarification; this kinda sounds like you are talking about how precisely the Ka'sei can be blown apart because they are sentient.

Reworded in the new commit, let me know if it's clearer

> Also, wouldn't some kind of automated defense system be more accurate than a sentient being?

For us, yeah, but the Builders' automated tech program came late and underperforming and so wasn't pursued as much as it could have been